### PR TITLE
Alerting plugin - experimental cross cluster monitor support documentation

### PR DESCRIPTION
**Pull request by AWSHurneyt**
### Description
Added documentation for supporting cluster metrics monitors that can monitor remote clusters. Added documentation for configuring query and bucket monitors through the UI that can query remote indexes. 

These are experimental for v2.12.

### Issues Resolved
Closes https://github.com/opensearch-project/documentation-website/issues/4438

### Checklist
- [x] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by AWSHurneyt** [comment]
@vagimeli Here's the PR for the documentation.

File: _observing-your-data/alerting/per-cluster-metrics-monitors.md [hunk 1]
```diff
@@ -68,7 +68,8 @@ Painless scripts define triggers for cluster metrics monitors, similar to per qu
 
 The cluster metrics monitor supports up to **ten** triggers.
 
-In the following example, a JSON object creates a trigger that sends an alert when the cluster health is yellow. `script` points the `source` to the Painless script `ctx.results[0].status == \"yellow\`.
+In the following example, the monitor is configured to call the cluster health API for two clusters, (`cluster-1`, and `cluster-2`). The trigger condition will create an alert when either of the clusters' `status` is not `green`.
```
Comment (vagimeli) [inline _observing-your-data/alerting/per-cluster-metrics-monitors.md:71]:
```suggestion
In the following example, the monitor is configured to call the Cluster Health API for two clusters, (`cluster-1` and `cluster-2`). The trigger condition will create an alert when either of the clusters' `status` is not `green`. 
```

File: _observing-your-data/alerting/per-query-bucket-monitors.md [hunk 2]
```diff
@@ -13,6 +13,10 @@ Per query monitors are a type of alert monitor that can be used to identify and
 
 Per bucket monitors are a type of alert monitor that can be used to identify and alert on specific buckets of data that are created by a query against an OpenSearch index.
 
+Both monitor types support querying remote indexes using the same `cluster-name:index-name` pattern used by [cross-cluster search](https://opensearch.org/docs/latest/security/access-control/cross-cluster-search/), or using the new dashboards interface.
```
Comment (vagimeli) [inline _observing-your-data/alerting/per-query-bucket-monitors.md:16]:
```suggestion
Both monitor types support querying remote indexes using the same `cluster-name:index-name` pattern used by [cross-cluster search](https://opensearch.org/docs/latest/security/access-control/cross-cluster-search/), or by using OpenSearch Dashboards 2.12 or later.
```

File: _observing-your-data/alerting/per-cluster-metrics-monitors.md [hunk 3]
```diff
@@ -68,7 +68,8 @@ Painless scripts define triggers for cluster metrics monitors, similar to per qu
 
 The cluster metrics monitor supports up to **ten** triggers.
 
-In the following example, a JSON object creates a trigger that sends an alert when the cluster health is yellow. `script` points the `source` to the Painless script `ctx.results[0].status == \"yellow\`.
+In the following example, the monitor is configured to call the cluster health API for two clusters, (`cluster-1`, and `cluster-2`). The trigger condition will create an alert when either of the clusters' `status` is not `green`. 
+`script` points the `source` to the Painless script `for (cluster in ctx.results[0].keySet()) if (ctx.results[0][cluster].status != \"green\") return true`. See [Trigger variables]({{site.url}}{{site.baseurl}}/observing-your-data/alerting/triggers/#trigger-variables) for more `painless ctx` variable options.
```
Comment (vagimeli) [inline _observing-your-data/alerting/per-cluster-metrics-monitors.md:72]:
```suggestion
The `script` parameter points the `source` to the Painless script `for (cluster in ctx.results[0].keySet()) if (ctx.results[0][cluster].status != \"green\") return true`. See [Trigger variables]({{site.url}}{{site.baseurl}}/observing-your-data/alerting/triggers/#trigger-variables) for more `painless ctx` variable options.
```

**Review by vagimeli — APPROVED** [review]
@AWSHurneyt Minimal edits. For the UI, please lowercase "Documentation" and remove the closing punctuation after "visit forum.opensearch.org".

**Comment by AWSHurneyt** [comment]
> @AWSHurneyt Minimal edits. For the UI, please lowercase "Documentation" and remove the closing punctuation after "visit forum.opensearch.org".

@vagimeli  Thank you! I've edited the banner accordingly.

File: _observing-your-data/alerting/per-cluster-metrics-monitors.md [hunk 4]
```diff
@@ -110,14 +112,14 @@ In the following example, a JSON object creates a trigger that sends an alert wh
   ]
 }
 ```
+The dashboards interface supports selecting clusters to monitor, and the desired API.
```
Comment (natebower) [inline _observing-your-data/alerting/per-cluster-metrics-monitors.md:115]:
```suggestion
The dashboards interface supports the selection of clusters to be monitored and the desired API.
```

File: _observing-your-data/alerting/per-query-bucket-monitors.md [hunk 5]
```diff
@@ -13,6 +13,10 @@ Per query monitors are a type of alert monitor that can be used to identify and
 
 Per bucket monitors are a type of alert monitor that can be used to identify and alert on specific buckets of data that are created by a query against an OpenSearch index.
 
+Both monitor types support querying remote indexes using the same `cluster-name:index-name` pattern used by [cross-cluster search](https://opensearch.org/docs/latest/security/access-control/cross-cluster-search/), or by using OpenSearch Dashboards 2.12 or later.
```
Comment (natebower) [inline _observing-your-data/alerting/per-query-bucket-monitors.md:16]:
```suggestion
Both monitor types support querying remote indexes using the same `cluster-name:index-name` pattern used by [cross-cluster search](https://opensearch.org/docs/latest/security/access-control/cross-cluster-search/) or by using OpenSearch Dashboards 2.12 or later.
```

File: _observing-your-data/alerting/per-query-bucket-monitors.md [hunk 6]
```diff
@@ -13,6 +13,10 @@ Per query monitors are a type of alert monitor that can be used to identify and
...
+
+<img src="{{site.url}}{{site.baseurl}}/images/alerting/cross-cluster-per-query-per-bucket-monitors.png" alt="Cluster metrics monitor" width="700"/>
```
Comment (natebower) [inline _observing-your-data/alerting/per-query-bucket-monitors.md:18]: The image should be introduced in the text that precedes it.

**Review by natebower — COMMENTED** [review]
@vagimeli Please see my comments and changes and let me know if you have any questions. Thanks!

**Pull request merged**